src/backend/opbeat_backend.js

```diff
@@ -133,6 +134,11 @@ OpbeatBackend.prototype.sendTransactions = function (transactionList) {
   if (this._config.isValid()) {
     var browserResponsivenessInterval = opbeatBackend._config.get('performance.browserResponsivenessInterval')
     var checkBrowserResponsiveness = opbeatBackend._config.get('performance.checkBrowserResponsiveness')
+    var pageLoad = opbeatBackend._config.get('performance.pageLoad')
+
+    if (pageLoad) {
+      transactionList.forEach(captureHardNavigation);
```

Comment:
`captureHardNavigation` looks for a property on the transaction to determine if it was the initial page load, so it's the callers responsibility to make sure only one of the transactions have that property